<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1124</width>
    <height>716</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayoutMain">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>2</number>
      </property>
      <widget class="QWidget" name="leftPanel">
       <layout class="QVBoxLayout" name="verticalLayoutLeft">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLineEdit" name="searchBar">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Enter a book title</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="bookList">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addBookButton">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Add New Book</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="cursor" stdset="0">
        <cursorShape>BusyCursor</cursorShape>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>96</x>
          <y>0</y>
          <width>500</width>
          <height>892</height>
         </rect>
        </property>
        <property name="minimumSize">
         <size>
          <width>400</width>
          <height>0</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayoutRight">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QLabel" name="coverLabel">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="autoFillBackground">
            <bool>true</bool>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item alignment="Qt::AlignRight">
          <widget class="QPushButton" name="changeCoverButton">
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>change cover</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="titleLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Title</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="descriptionEdit">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="ratingLabel">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Rating</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ratingEdit">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="ReviewLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Review</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="reviewEdit">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Review</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="noteLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Note</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="noteEdit">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="cursor" stdset="0">
            <cursorShape>ArrowCursor</cursorShape>
           </property>
           <property name="placeholderText">
            <string>Note</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="commentLabel">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>TextLabel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="commentEdit">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="placeholderText">
            <string>Comment</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="isReadingBox">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>I read this book</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="startDateLabel">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Reading start date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="startDateEdit">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="finishDateLabel_2">
             <property name="font">
              <font>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Reading finish date</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDateEdit" name="finishDateEdit_2">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="editButton">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Edit Info</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="deleteBookButton">
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Delete book</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1124</width>
     <height>19</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <widget class="QMenu" name="menuLog_out">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuSort_2">
     <property name="title">
      <string>Sort</string>
     </property>
     <addaction name="actionSort_by_name"/>
     <addaction name="actionBy_Rating"/>
     <addaction name="actionBy_start_date_of_reading_3"/>
     <addaction name="actionBy_date_of_completion_of_the_reading_2"/>
    </widget>
    <addaction name="actionNew_book"/>
    <addaction name="separator"/>
    <addaction name="menuSort_2"/>
    <addaction name="separator"/>
    <addaction name="actionLog_out"/>
   </widget>
   <widget class="QMenu" name="menuBackup">
    <property name="title">
     <string>Backup</string>
    </property>
    <addaction name="actionCreate_backup"/>
    <addaction name="actionRestore_backup"/>
   </widget>
   <addaction name="menuLog_out"/>
   <addaction name="menuBackup"/>
  </widget>
  <action name="actionBy_name">
   <property name="text">
    <string>By name</string>
   </property>
  </action>
  <action name="actionBy_start_date_of_reading">
   <property name="text">
    <string>By start date of reading </string>
   </property>
  </action>
  <action name="actionBy_date_of_completion_of_the_reading">
   <property name="text">
    <string>By date of completion of the reading</string>
   </property>
  </action>
  <action name="actionCreate_backup">
   <property name="text">
    <string>Create backup</string>
   </property>
  </action>
  <action name="actionRestore_backup">
   <property name="text">
    <string>Restore backup</string>
   </property>
  </action>
  <action name="actionNew_book">
   <property name="text">
    <string>New book</string>
   </property>
  </action>
  <action name="actionSort_by_name">
   <property name="text">
    <string>By name</string>
   </property>
  </action>
  <action name="actionBy_start_date_of_reading_2">
   <property name="text">
    <string>By start date of reading </string>
   </property>
  </action>
  <action name="actionBy_date_of_completion_of_the_reading_2">
   <property name="text">
    <string>By date of completion of the reading</string>
   </property>
  </action>
  <action name="actionLog_out">
   <property name="text">
    <string>Log out</string>
   </property>
  </action>
  <action name="actionBy_Rating">
   <property name="text">
    <string>By Rating</string>
   </property>
  </action>
  <action name="actionBy_start_date_of_reading_3">
   <property name="text">
    <string>By start date of reading </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
